src/sigi/database/dao/AccountingAccountsDao.java

```diff
@@ -0,0 +1,69 @@
+package sigi.database.dao;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.ArrayList;
+
+
+import sigi.database.dto.AccountingAccountsDto;
+import sigi.database.lib.IDataSource;
+
+public class AccountingAccountsDao {
+    private IDataSource dataSource = null;
+
+    public AccountingAccountsDao(IDataSource ds){
+        this.dataSource = ds;
+    }
+
+    public void setDataSource(IDataSource ds){
+        this.dataSource = ds;
+    }
+
+    public ArrayList<AccountingAccountsDto>getListAccountingA(){
+        ArrayList<AccountingAccountsDto> accounting = new ArrayList<AccountingAccountsDto>();
+        
+        try{
+                String sql = "Select TOP 20  Name, Description, Status, AccountType from AccountingAccounts where  AccountType = '2' ORDER BY Id DESC";
```

Comment:
String[] codes = String[6]{"777","555","555"};
for(String c: codes){
String sql = "Select TOP 20  Name, Description, Status, AccountType from AccountingAccounts where  code = "+ c +" ORDER BY Id DESC";
}